src/types/Health.ts

```diff
@@ -336,30 +410,40 @@ export class WorkloadHealth extends Health {
     }
     // Request errors
     if (ctx.hasSidecar) {
-      const reqIn = getRequestErrorsStatus(requests.inboundErrorRatio);
-      const reqOut = getRequestErrorsStatus(requests.outboundErrorRatio);
+      const reqError = calculateErrorRate(ns, workload, 'workload', requests);
+      const reqIn = reqError.errorRatio.inbound.status;
+      const reqOut = reqError.errorRatio.outbound.status;
       const both = mergeStatus(reqIn.status, reqOut.status);
       const item: HealthItem = {
         title: 'Error Rate over ' + getName(ctx.rateInterval).toLowerCase(),
         status: both,
         children: [getRequestErrorsSubItem(reqIn, 'Inbound'), getRequestErrorsSubItem(reqOut, 'Outbound')]
       };
       items.push(item);
+
+      statusConfig = {
+        title: 'Error Rate over ' + getName(ctx.rateInterval).toLowerCase(),
```

Comment:
I guess this same title is repeated 5 or so times in various places, perhaps it needs a small utility function to centralize and ensure consistency.